core/src/classes/configTemplate.ts

```diff
@@ -49,26 +52,28 @@ export abstract class ConfigTemplate {
   }): Promise<ConfigTemplateRunResponse>;
 
   prepareParams(params: ConfigTemplateParams) {
+    this.params = params;
```

Comment:
Nice.   I'll bet this can simplify some of the usage in the `bin/generate` command too

Reply:
🎉  I was thinking it could come in handy down the road without affecting any of the existing code today!